src/modules/ContactPage/components/molecules/ContactUsForm/index.tsx

```diff
@@ -0,0 +1,158 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Button, InputField, SelectInput } from 'm3ntorship-ui';
+import { ReactElement, useEffect, useState } from 'react';
+import { Controller, SubmitHandler, useForm } from 'react-hook-form';
+import * as Yup from 'yup';
+
+import ApplicationSuccess from '@/modules/ApplicationsPage/components/molecules/ApplicationSuccess';
+
+const schema = Yup.object({
+  subject: Yup.string().required(),
+  name: Yup.string().required().min(4, 'Min chacracter length is 4'),
+  email: Yup.string().required().email(),
+  message: Yup.string().required().min(10, 'Min chacracter length is 10'),
+}).required();
+
+type IFormFields = {
+  subject: string;
+  name: string;
+  email: string;
+  message: string;
+};
+const ContactUsForm = (): ReactElement => {
+  const {
+    handleSubmit,
+    reset,
+    control,
+    formState,
+    formState: { errors },
+    formState: { isSubmitSuccessful },
+  } = useForm<IFormFields>({
+    mode: 'onChange',
+    resolver: yupResolver(schema),
+  });
+  const [dataSubmited, setSubmitedData] = useState(false);
+  const onSubmit = (data: IFormFields, e: React.MouseEvent) => {
+    setSubmitedData(true);
+    e.preventDefault();
+  };
+  useEffect(() => {
+    if (isSubmitSuccessful) {
+      reset({
+        subject: '',
+        name: '',
+        email: '',
+        message: '',
+      });
+    }
+  }, [reset, formState, isSubmitSuccessful]);
+  return (
+    <form
+      role='form'
+      name='contact us form'
+      onSubmit={handleSubmit(onSubmit as SubmitHandler<IFormFields>)}
+      className='flex border-spacing-2 flex-col gap-y-6 rounded-lrg bg-white p-8 shadow-shd-1'
```

Comment:
form padding should be `40px` not `32px` 
```suggestion
      className='flex border-spacing-2 flex-col gap-y-6 rounded-lrg bg-white p-10 shadow-shd-1'
```